c.html.markdown

```diff
@@ -96,13 +112,14 @@ int main (int argc, char** argv)
   // For the sake of the tutorial, variables are declared dynamically under
   // C99-compliant standards.
 
-  // ints are usually 4 bytes
+  // ints are usually 4 bytes (use the 'sizeof' operator to check)
```

Comment:
```suggestion
  // ints are usually 4 bytes (use the `sizeof` operator to check)
```

Reply:
i didn't realize that markdown would format code fences... inside of a code fence.

Reply:
I actually don't know it for sure, but it's easier to discern the code that way, on my opinion :)